pyproject.toml

```diff
@@ -10,6 +10,8 @@ home-page = "https://github.com/takluyver/pep517"
 description-file = "README.rst"
 requires = [
     "pytoml",
+    "importlib_metadata",
```

Comment:
why is this a build dependency? 🤔

Reply:
I thought that was an install dependency.

Reply:
This one's the flit metadata section, not the build-system section.

It's a bit unfortunate the name's used in 2 places for 2 different things :-(